website/content/docs/commands/job/action.mdx

```diff
@@ -0,0 +1,109 @@
+in Nomad job specifications. These actions can be defined at task level and are
+intended for specific operational tasks, such as clearing a cache, or migrating
+a database.
+
+An action may self-terminate upon completion (for example, echoing a string),
+or run for an indeterminate amount of time (for example, watching a blocking
+query). In the latter case, an action can be terminated via escape character
+(such as cmd+c / ctrl+c)
+
+When ACLs are enabled, this command requires a token with the `alloc-exec`,
+`read-job`, and `list-jobs` capabilities for the allocation's namespace. If
+the task driver does not have file system isolation (as with `raw_exec`),
+this command requires the `alloc-node-exec`, `alloc-exec`, `read-job`, and
+`list-jobs` capabilities for the allocation's namespace.
+
+## Usage
+
+```plaintext
+nomad job action [options]
+```
+
+The `job action` command requires an action name and accepts two ways of
+specifying where it should be run:
+- by passing the job name and known allocation ID along with the name of the action
+- by passing the job, task group, and task name along with the name of the action
+(when the allocation ID is not known). A random allocation will be selected if
+multiple are available.
+
+The action name provided must be defined within a task in the [job specification]
+provided. With sufficient privileges, an execution context will be opened and
+the defined action command will be run. No further input is possible, save for
+the escape character to terminate execution, so interactive commands are not
+supported.
+
+## General Options
+
+@include 'general_options.mdx'
+
+## Action Options
+
+- `-job`: (Required) Specifies the job containing the predefined action. This
+  is the job under which the task is registered.
+
+- `-alloc`: Specifies the allocation within which the action is to be executed.
+  If omitted, either a task-group and task-name must be provided, and a random
+  allocation will be selected from them.
+
+- `-task`: (Required if no allocation is specified) Specifies the task within
+  the job where the action is defined. This must correspond to a task in the job's
+  specification.
+
+- `-group`: (Required if no allocation is specified) Specifies the task group 
+  within the job. This is needed when selecting a random allocation from a
+  specific task.
+
+- `-i`: Pass stdin to the container, defaults to true. Pass `-i=false` to
+  disable explicitly.
+
+- `-t`: Allocate a pseudo-tty, defaults to true if stdin is detected to be a tty
+  session. Pass `-t=false` to disable explicitly.
+
+- `-e` <escape_char>: Sets the escape character for sessions with a pty
+  (default: '~'). The escape character is only recognized at the beginning of a
+  line. The escape character followed by a dot ('.') closes the connection.
+  Setting the character to 'none' disables any escapes and makes the session
+  fully transparent.
```

Comment:
I think we tend to state whether they are required or not as part of the normal sentence which is also my personal preference; but not a hard requirement.